Source/WebCore/css/parser/CSSPropertyParserConsumer+Grid.h

```diff
@@ -47,13 +50,13 @@ enum TrackListType : uint8_t { GridTemplate, GridTemplateNoRepeat, GridAuto };
 
 bool isGridBreadthIdent(CSSValueID);
 bool parseGridTemplateAreasRow(StringView gridRowNames, NamedGridAreaMap&, const size_t rowCount, size_t& columnCount);
```

Comment:
Spotted this in passing:
```suggestion
bool parseGridTemplateAreasRow(StringView gridRowNames, NamedGridAreaMap&, size_t rowCount, size_t& columnCount);
```